hoomd/hpmc/IntegratorHPMC.h

```diff
@@ -260,6 +261,31 @@ class PYBIND11_EXPORT IntegratorHPMC : public Integrator
         return m_nselect;
         }
 
+    //! Set kT variant
+    /*! \param kT new k_BT variant to set
+     */
+    void setkT(std::shared_ptr<Variant> kT)
+        {
+        m_kT = kT;
+        }
+
+    //! Get kT variant
+    //! \returns current value of kT parameter
+    std::shared_ptr<Variant> getkT()
+        {
+        return m_kT;
+        }
```

Comment:
```suggestion
    std::shared_ptr<const Variant> getKT()
        {
        return m_kT;
        }
```
The capital K is necessary other wise the first word of the method name appears to be `getk` and the second word as `T`.